src/PostgREST/ApiRequest/Preferences.hs

```diff
@@ -49,15 +51,17 @@ data Preferences
     , preferTransaction    :: Maybe PreferTransaction
     , preferMissing        :: Maybe PreferMissing
     , preferHandling       :: Maybe PreferHandling
+    , preferTimezone       :: Maybe ByteString
```

Comment:
@taimoorzaeem It'd be more consistent to turn this into a `Maybe PreferTimezone`. Then we can have:

```hs
newtype PreferTimezone = PreferTimezone ByteString

instance ToHeaderValue PreferTimezone where
  toHeaderValue (PreferTimezone bs) = "timezone=" <> bs
```